src/foam/u2/wizard/wizardlet/AlternateFlowWizardlet.js

```diff
@@ -37,6 +46,9 @@ foam.CLASS({
           this.AlternateFlowAction.create({ alternateFlow }));
         this.dynamicActions = this.dynamicActions.concat(choices);
       }
+      if ( ! this.AlternateFlowWAO.isInstance(this.wao) ) {
+        this.wao = this.AlternateFlowWAO.create({ delegate: this.wao }, this.__subContext__);
+      }
```

Comment:
So AlternateFlowWizardlet will always have this behaviour? Usually I expect alternate flow actions to save the wizardlet before proceeding, so I'd say this is an exception to the normal behaviour.

Reply:
updated